tests/lammps/test_base.py

```diff
@@ -678,3 +678,31 @@ def test_units(self):
         def setter(x):
             self.job.units = x
         self.assertRaises(ValueError, setter, "nonsense")
+
+    def test_bonds_input(self):
+        potential = pd.DataFrame({'Name': ['Morse'],
+                                  'Filename': [[]],
+                                  'Model'   : ['Morse'],
+                                  'Species' : [['Al']],
+                                  'Config'  : [['atom_style bond\n',
+                                                'bond_style morse\n',
+                                                'bond_coeff 1 0.1 1.5 2.0\n',
+                                                'bond_coeff 2 0.1 1.5 2.0']]})
+        cell = Atoms(elements=4*['Al'], positions=[[0., 0., 0.],
+                                                   [0., 2., 2.],
+                                                   [2., 0., 2.],
+                                                   [2., 2., 0.]],
+                     cell=4*np.eye(3))
+        self.job.structure = cell.repeat(2)
+        self.job.structure.bonds = [[1, 2, 1], [1, 3, 2]]
+        self.job.potential = potential
+        self.job.calc_static()
+        self.job.run(run_mode="manual")
+
+        bond_str = "2 bond types\n"
+        struct_file = open(self.job.get_workdir_file('structure.inp'), 'r')
+        struct_lines = struct_file.readlines()[4]
+        struct_file.close()
+        self.assertTrue(struct_lines[-1], bond_str)
```

Comment:
This can be simplified 

```suggestion
        self.assertTrue(self.job["structure.inp"][4][-1], bond_str)
```

Reply:
Awesome! I didn't know I could access the input files this way! :D